maquetado/scss/tech.scss

```diff
@@ -25,22 +25,22 @@
         z-index: -1;
     }
 
-    .tech-logo{
+    .tech-logo {
         width: 100px;
```

Comment:
Add the variable `$tech-logo-size: 100px;` previously presented  in the setup trello card and use it here.